src/main/java/core/basesyntax/FigureInfoDisplay.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface FigureInfoDisplay {
```

Comment:
Can't we use it for Machine or Animal class?
```suggestion
public interface Drawable {
```